### PR TITLE
[FIX] pylint_odoo: Check if the package start with dot to skip 

### PR DESCRIPTION
**Pull request by JesusZapata**
Check if the package start with dot to skip because this package is absolute import

This change now detect when the import start with dot for example

```python
from ...no_exists_one import package_one
from .no_exists_tow import package_two
from . import package_three
from .. import package_four
```
If the import start with dot that import is not detect as external dependency

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
Your opinion is welcome

File: pylint_odoo/checkers/modules_odoo.py [hunk 1]
```diff
@@ -360,6 +360,8 @@ def _is_absolute_import(node, name):
                 modnode is not importedmodnode and \
                 importedmodnode.name != name:
             return True
+        if node.as_string().split()[1].startswith('.'):
```
Comment (moylop260) [inline pylint_odoo/checkers/modules_odoo.py]: Is a absolute import If starts with `.` ?

Comment (JesusZapata) [reply pylint_odoo/checkers/modules_odoo.py]: @moylop260 
I change this if statement to another location in favor to keep the `method _is_absolute_import` without change

File: pylint_odoo/checkers/modules_odoo.py [hunk 2]
```diff
@@ -403,7 +403,8 @@ def _check_imported_packages(self, node, module_name):
 
         ext_deps = self.manifest_dict.get('external_dependencies') or {}
         py_ext_deps = ext_deps.get('python') or []
-        if module_name not in py_ext_deps and \
+        if node.as_string().split()[1].startswith('.') and \
```
Comment (moylop260) [inline pylint_odoo/checkers/modules_odoo.py]: Is possible a IndexError here?

Could you run this version in all your python files locally?

Comment (JesusZapata) [reply pylint_odoo/checkers/modules_odoo.py]: First run `pylint_odoo` over the code without this modification 
`git checkout master`

```bash
(py34-pylint20) [odoo@73f00e6f2458]~/build/Vauxoo/pylint-odoo$
 (master)$ python -m pylint -d all -e missing-manifest-dependency --load-plugins=pylint_odoo /home/odoo/build/mexico/l10n_mx_landing
No config file found, using default configuration
************* Module l10n_mx_landing.tests.test_landing
W: 10, 0: Missing external dependency "l10n_mx_base.tests.common" from manifest. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies (missing-manifest-dependency)

------------------------------------------------------------------
Your code has been rated at 9.89/10 (previous run: 9.89/10, +0.00)
```

And after run `pylint_odoo` over the code with this modification 
`git checkout master-vx-fix-external-dependency-jesuszapata`

```bash
(py34-pylint20) [odoo@73f00e6f2458]~/build/Vauxoo/pylint-odoo$
 (master-vx-fix-external-dependency-jesuszapata)$ python -m pylint -d all -e missing-manifest-dependency --load-plugins=pylint_odoo /home/odoo/build/mexico/l10n_mx_landing
No config file found, using default configuration

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

**Comment by JesusZapata** [comment]
@moylop260 
The travis is red because a not related check 

```bash
flake8 --ignore=F601 --exclude=__init__.py .
./pylint_odoo/checkers/modules_odoo.py:369:9: E722 do not use bare except'
The command "flake8 --ignore=F601 --exclude=__init__.py ." exited with 1.
```
The `pylint_odoo` test ran well

**Comment by JesusZapata** [comment]
@moylop260 
Related with my previous comment  the same problem was solved in  https://github.com/OCA/pylint-odoo/pull/166
I created this pull request to get all change in OCA to Vauxoo fork https://github.com/Vauxoo/pylint-odoo/pull/135

**Comment by JesusZapata** [comment]
@moylop260 
This pull request now is green

File: pylint_odoo/checkers/modules_odoo.py [hunk 3]
```diff
@@ -413,7 +413,8 @@ def _check_imported_packages(self, node, module_name):
 
         ext_deps = self.manifest_dict.get('external_dependencies') or {}
         py_ext_deps = ext_deps.get('python') or []
-        if module_name not in py_ext_deps and \
+        if not node.as_string().split()[1].startswith('.') and \
```
Comment (moylop260) [inline pylint_odoo/checkers/modules_odoo.py]: You can use the same way from https://github.com/PyCQA/pylint/pull/888#r62025010

If is instance of `astroid.ImportFrom` class then you can use `node.level >= 1` in order to detect a `from . import package`

Comment (JesusZapata) [reply pylint_odoo/checkers/modules_odoo.py]: @moylop260 
Thanks for your help, now I used the node.level to avoid the node.as_string

**Comment by JesusZapata** [comment]
@moylop260 
This is the pull request https://github.com/OCA/pylint-odoo/pull/171

**Comment by moylop260** [comment]
Please, now you could update vx/pylint from oca/pylint

**Pull request closed without merging**